### ISSUE TITLE
lab 5- Step1

### ISSUE THREAD

**Issue by Jana-Ajeeb** [Watts-College/cpp-527-fall-2021#39]
Hello,

I'm trying to set the branch as master branch but i couldn't find it.
![branch](https://user-images.githubusercontent.com/77167811/133944861-9efd171e-1471-4fe6-9e23-32706f840078.PNG)

Can you please help in what can i do here?

**Comment by lecy** [comment]
They changed it to “main” branch now.

**Comment by ndavis4904** [comment]
Mine says "root", is that the same thing?

**Comment by lecy** [comment]
@ndavis4904 strange, is it the default option under the GitHub Pages tab? It might be but I’ve not seen that before. There is a root branch in some repos, though.

**Comment by Jana-Ajeeb** [comment]
I tried the main and roots but i guess it's saving as a file on my PC not as site. So, maybe I should change my branch or something like that?
![11](https://user-images.githubusercontent.com/77167811/134038503-143ebbe0-f43d-4fd9-a908-56f83c8acf89.PNG)

**Comment by lecy** [comment]
@Jana-Ajeeb are you looking under **Settings>>Page**?

**Comment by Jana-Ajeeb** [comment]
yes In the Settings>>Page then in the branches their is just "main" option
![12](https://user-images.githubusercontent.com/77167811/134049970-1d0e5c63-4100-4555-8fec-40b4902120ab.PNG)
 
then 2 options appear either doc or roots where both lead to the same result which is saving on the PC not creating a site :
![1111](https://user-images.githubusercontent.com/77167811/134050298-fd58aa09-68b7-449f-82d0-fda153d97430.PNG)

**Comment by lecy** [comment]
Gotcha, you need to select the root folder. That just means not selecting a subfolder (sometimes sites place all HTML files in a separate folder like /docs). Root is the base directory for main.

**Comment by lecy** [comment]
Did you try the link in the green box that says: "Your site is published at... URL" ?

**Comment by Jana-Ajeeb** [comment]
Yes i tried it :
![113](https://user-images.githubusercontent.com/77167811/134059627-f73a8387-4a71-47e7-bb70-fd45ef0b550b.PNG)

But I'm not being able to continue after this step normally

**Comment by lecy** [comment]
Did you select a Theme option on the Pages setting? It looks like you did in this screen shot. 

I just looked at your page and it looks correct now?

**Comment by Jana-Ajeeb** [comment]
hello,

Actually I was mixing up between opening a whole new site and cloning another one and editing it.
However, now I got it and it's working so far.

Thanks dr.
